### PR TITLE
Make for better autocomplete experience

### PR DESCRIPTION
**Pull request by benogle**
* [x] Add config for symbol provider
* [x] Add less [builtin functions](http://lesscss.org/functions/)

### CODE REVIEW

File: update.coffee [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Run this to update the list of builtin less functions
+
+path = require 'path'
+fs = require 'fs'
+request = require 'request'
+Promise = require 'bluebird'
```
Comment (thomasjo) [inline update.coffee]: Why is this needed? We have native `Promise` now.

Comment (benogle) [inline update.coffee]: This file is not run in atom's context, and when run from node on the command line, it's not there :( Maybe I'm doing it wrong though.
